subsys/net/lib/aws_fota/src/aws_fota_json.c

```diff
@@ -167,37 +153,58 @@ int aws_fota_parse_update_job_exec_state_rsp(char *update_rsp_document,
 	return ret;
 }
 
-int aws_fota_parse_notify_next_document(char *job_document,
-		u32_t payload_len, char *job_id_buf, char *hostname_buf,
-		char *file_path_buf)
+#define EXECUTION_OBJ_DECODED_BIT 2
```

Comment:
Add define for 0x01 as well (used above)

Reply:
I suggest moving the defines to aws_fota_json.h (add a short description as well)

Reply:
Done.